pyro/distributions/torch_patch.py

```diff
@@ -218,15 +55,6 @@ def _einsum(equation, operands):
         y, x = operands
         return (x.unsqueeze(1) * y).sum(0).transpose(0, 1)
 
-    # work around torch.einsum's limitation to 26 letters
-    symbols = sorted(set(equation) - set(',->'))
```

Comment:
We unfortunately still need this to avoid `RuntimeError: only lowercase letters a-z allowed as indices`.

Reply:
Hmm I thought I moved that logic upstream to `opt_einsum` [here](https://github.com/dgasmith/opt_einsum/blob/3d72bba/opt_einsum/parser.py#L88) and [here](https://github.com/dgasmith/opt_einsum/blob/3d72bba4087a56411b19ce30c1a68a70773113dd/opt_einsum/backends/torch.py#L41). Where are you seeing errors? Maybe it's missing from our `torch_log` backend.